.github/CODEOWNERS

```diff
@@ -103,6 +103,7 @@
 /src/material-experimental/mdc-progress-bar/**     @andrewseguin
 # Note to implementer: please repossess
 /src/material-experimental/mdc-radio/**            @mmalerba
+/src/material-experimental/mdc-snackbar/**         @opozo
```

Comment:
In your next PR, you can add me here as well

Reply:
Ack.